src/components/PeopleTable/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,22 @@
+import { FC } from 'react';
+import { NavLink } from 'react-router-dom';
+import cn from 'classnames';
+import { Person } from '../../../types/Person';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: FC<Props> = ({ person }) => {
+  const { sex, slug, name } = person;
+
+  return (
+
+    <NavLink
+      to={`/people/${slug}`}
+      className={cn({ 'has-text-danger': sex === 'f' })}
```

Comment:
would be really cool to create a variable outside the component for "f" because now that's the magic value

```jsx
const FEMALE = 'f';
```